task3/Ex6.Rmd

```diff
@@ -12,8 +12,20 @@ if (!("RPostgreSQL" %in% installed.packages())){
 if (!("getPass" %in% installed.packages())){
   install.packages("getPass")
 }
+if (!("tmap" %in% installed.packages())){
+  install.packages("tmap")

```

Comment:
For me it was necessary to install tmap and XML manually. You could leave that as a comment in the code.

Reply:
It was the same for me! I even had to delete a folder in my packages directory, because this threw an error somehow.